<launch>
    <!-- This version uses the openai_ros environments -->
    <rosparam command="load" file="$(find turtlebot2_maze_env)/config/turtlebot2_maze_params.yaml" />
    <!-- Launch the training system -->
    <node pkg="turtlebot2_maze_env" name="turtlebot2_maze_random" type="turtlebot2_maze_random.py" output="screen"/>
</launch>
